openssl/src/ec.rs

```diff
@@ -477,6 +477,32 @@ impl EcPointRef {
         }
     }
 
+    /// Place affine coordinates of a curve over a prime field in the provided
+    /// `x` and `y` `BigNum`s
+    ///
+    /// OpenSSL documentation at [`EC_POINT_get_affine_coordinates`]
+    ///
+    /// [`EC_POINT_get_affine_coordinates`]: https://www.openssl.org/docs/man1.1.1/man3/EC_POINT_get_affine_coordinates.html
+    #[cfg(ossl111)]
+    pub fn affine_coordinates(
+        &self,
+        group: &EcGroupRef,
+        x: &mut BigNumRef,
```

Comment:
Passing in these pointers will cause memory leaks, since EC_POINT_get_affine_transformation does not free the old value. The arguments are intended to be used as out-pointers, where this method would return `Result<(&BigNumRef, &BigNumRef), ErrorStack>`.

Reply:
Oh wait no I'm wrong, sorry.